.gitignore

```diff
@@ -18,7 +18,7 @@
 # The .vscode folder contains launch configuration and tasks you configure in
 # VS Code which you may wish to be included in version control, so this line
 # is commented out by default.
-#.vscode/
+.vscode/
```

Comment:
Are there some files that we want to remove from source control along with this change?

Reply:
Nope